src/tag-editor/utils.ts

```diff
@@ -46,3 +50,46 @@ export function useMemoizedArray<T>(array: ReadonlyArray<T>, isEqual: (prev: T,
   }, [updated]);
   return updated;
 }
+
+interface GetTagsDiffResult {
+  createdTags: Record<string, string>;
```

Comment:
[minor] I would suggest to rename these props to just `created` and `removed`, since the function name already has `tags` in the name

Reply:
updated - good call